docs/reference/cat/anomaly-detectors.asciidoc

```diff
@@ -0,0 +1,281 @@
+
+`bucket.time.exp_avg`, `btea`, `bucketTimeExpAvg`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-time-exponential-average]
+
+`bucket.time.exp_avg_hour`, `bteah`, `bucketTimeExpAvgHour`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-time-exponential-average-hour]
+
+`bucket.time.max`, `btmax`, `bucketTimeMax`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-time-maximum]
+
+`bucket.time.min`, `btmin`, `bucketTimeMin`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-time-minimum]
+
+`bucket.time.total`, `btt`, `bucketTimeTotal`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-time-total]
+
+`data.buckets`, `db`, `dataBuckets`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-count]
+
+`data.earliest_record`, `der`, `dataEarliestRecord`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=earliest-record-timestamp]
+
+`data.empty_buckets`, `deb`, `dataEmptyBuckets`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=empty-bucket-count]
+
+`data.input_bytes`, `dib`, `dataInputBytes`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=input-bytes]
+
+`data.input_fields`, `dif`, `dataInputFields`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=input-field-count]
+
+`data.input_records`, `dir`, `dataInputRecords`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=input-record-count]
+
+`data.invalid_dates`, `did`, `dataInvalidDates`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=invalid-date-count]
+
+`data.last`, `dl`, `dataLast`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=last-data-time]
+
+`data.last_empty_bucket`, `dleb`, `dataLastEmptyBucket`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=latest-empty-bucket-timestamp]
+
+`data.last_sparse_bucket`, `dlsb`, `dataLastSparseBucket`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=latest-sparse-record-timestamp]
+
+`data.latest_record`, `dlr`, `dataLatestRecord`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=latest-record-timestamp]
+
+`data.missing_fields`, `dmf`, `dataMissingFields`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=missing-field-count]
+
+`data.out_of_order_timestamps`, `doot`, `dataOutOfOrderTimestamps`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=out-of-order-timestamp-count]
+
+`data.processed_fields`, `dpr`, `dataProcessedFields`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=processed-field-count]
+
+`data.processed_records`, `dpr`, `dataProcessedRecords`:::
+(Default)
+include::{docdir}/ml/ml-shared.asciidoc[tag=processed-record-count]
+
+`data.sparse_buckets`, `dsb`, `dataSparseBuckets`:::
+include::{docdir}/ml/ml-shared.asciidoc[tag=sparse-bucket-count]
+
+`forecast.memory.avg`, `fmavg`, `forecastMemoryAvg`:::
+The average memory usage in bytes for forecasts related to the {anomaly-job}.
+  
+`forecast.memory.max`, `fmmax`, `forecastsMemoryMax`:::
+The maximum memory usage in bytes for forecasts related to the {anomaly-job}.
+//TBD: Is there a reason why this "forecasts" is plural?
```

Comment:
`forecasts` is referring to ALL forecasts that this job has. This returns the maximum memory used by a single forecast of all the forecasts. 

Feel free to fix my unremarkable writing :)

Reply:
In that case, should all the others (e.g. "forecastMemoryMin") be plural too?